cnv/cnv_users_guide/cnv-using-the-default-pod-network-with-cnv.adoc

```diff
@@ -0,0 +1,31 @@
+[id="cnv-using-the-default-pod-network-with-cnv"]
+= Using the default Pod network with {ProductName}
+include::modules/cnv-document-attributes.adoc[]
+:context: cnv-using-the-default-pod-network-with-cnv
+toc::[]
+
+You can use the default Pod network with {ProductName}. To do so,
+you must use the `masquerade` binding method. It is the only recommended
+binding method for use with the default Pod network. Do not use
```

Comment:
Just a suggestion - to me the sentence beginning with, "It is the only recommended binding method..." feels redundant, as you've just said that this method must be used with the default Pod network.

Reply:
You're not wrong :) I was trying to really drive the point home because there are issues if you use `bridge` instead. It'll be all over the release notes, though, so I probably don't have to be so redundant.

Reply:
Fair enough. We can leave it as is for now.